### PR TITLE
impr(oracledb): 4 fix

### PR DESCRIPTION
**Pull request by maximelkin**
Idk how to test different oracledb packages
also maybe need to add some tests?

### CODE REVIEW

**Comment by kibertoad** [comment]
Travis needs to be updated to use latest driver, that would work as a test.

**Comment by kibertoad** [comment]
Fixes https://github.com/knex/knex/pull/3480

**Comment by kibertoad** [comment]
@maximelkin You can remove version from `https://github.com/knex/knex/blob/b0c0db601a67e67e0579f84892d8f07890abe9a3/.travis.yml#L19` and that should suffice.

File: .travis.yml [hunk 1]
```diff
@@ -16,7 +16,6 @@ matrix:
       env: TESTSCRIPT=test:everything DB="oracledb mssql mysql mysql2 postgres sqlite3" KNEX_TEST_TIMEOUT=60000
 install:
   - npm i
-  - (echo $DB | grep oracledb) && npm install oracledb@3.1.2 || true
```
Comment (kibertoad) [inline .travis.yml]: wait, whole line shouldn't be removed :D. just the `@3.1.2` part, so that latest driver version is picked up.

**Comment by kibertoad** [comment]
@maximelkin Is anything left on this PR? (you kept it asWIP)

**Comment by maximelkin** [comment]
I just want check existence of tests for lobs

**Comment by kibertoad** [comment]
Thanks!

**Comment by kibertoad** [comment]
@maximelkin Thank you for your most impressive contributions lately. Would you be interested in becoming one of the project maintainers?

**Comment by maximelkin** [comment]
@kibertoad 
>Would you be interested in becoming one of the project maintainers?


Sorry, not interested

**Comment by MaliaGuerrero** [comment]
Hi - there's a bug with this.  If you are reading from a BLOB, this function fails:

`function readStream(stream, type)`

You need to set data to a Buffer:

```
    let data = '';
    if (type === 'buffer') {
      data = Buffer.alloc(0);
    }
```

I'm trying to do a PR, but I can't push....so struggling with that.

**Comment by maximelkin** [comment]
@MaliaGuerrero fork repo, push to it, then create PR from your fork to this repo

**Comment by MaliaGuerrero** [comment]
Yeah, I'm getting a 403 error.  I'm attempting to work through it.

**Comment by kibertoad** [comment]
@MaliaGuerrero I think you are just trying to clone it, as I don't see you having created personal fork. Check this: https://help.github.com/en/github/getting-started-with-github/fork-a-repo

**Comment by MaliaGuerrero** [comment]
Thanks!  That fixed it.  I made a PR.

**Pull request merged**